colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -138,7 +190,7 @@ class HttpHeaders(private val headers: Array[HttpHeader]) {
     case other => false
   }
 
-  override def toString = "[" + headers.map{_.toString}.mkString(" ") + "]"
+  override def toString = "[" + toSeq.map{_.toString}.mkString(" ") + "]"
```

Comment:
super minor comment, but if this is called on one of those hot paths, this iterates 2x :).  Probably doesn't matter too much, but in the spirit of the MTA, "if you see something say something" :-p